rwengine/src/engine/GameWorld.cpp

```diff
@@ -750,15 +751,14 @@ void GameWorld::PhysicsTickCallback(btDynamicsWorld* physWorld,
 void GameWorld::loadCutscene(const std::string& name) {
     auto datfile = data->index.openFile(name + ".dat");
 
-    CutsceneData* cutscene = new CutsceneData;
+    auto cutscene = CutsceneData();
```

Comment:
Curly braces?

Reply:
It shouldn't make any difference. I'll try to avoid moving. ;)